static/css/styles.scss

```diff
@@ -348,13 +344,12 @@ a.subheading-link {
   padding-right: 15px;
   vertical-align: middle;
   background-image: none;
-  font-size: 12pt;
```

Comment:
This is also defined below on L359 (or L354 -- depending on if you're looking at unified or split diff).